plugins/modules/cloud/misc/terraform.py

```diff
@@ -80,9 +80,24 @@
     aliases: [ 'variables_file' ]
   variables:
     description:
-      - A group of key-values to override template variables or those in
-        variables files.
+      - A group of key-values pairs to override template variables or those in variables files.
+        By default, only string, boolean, and number values are allowed, which are passed on unquoted.
```

Comment:
```suggestion
        By default, only string and number values are allowed, which are passed on unquoted.
```
In that case booleans should also be removed from this one.